src/scrape_up/codeforces/user.py

```diff
@@ -58,105 +51,34 @@ def get_user_data(self):
-
-            location_info = info_list[2].split("From")
-            user_info["country"] = location_info[0].strip()
-            user_info["organization"] = location_info[1]
-
-            # Extract user's rating and max rating
-            user_rating = soup.find("div", class_="info").find_all("li")
-            for item in user_rating:
-                if "Contest rating:" in item.text:
-                    rating_span = item.find("span", class_="user-legendary")
-                    user_info["rating"] = (
-                        self._extract_text_or_empty(rating_span) if rating_span else ""
-                    )
-                elif "max. legendary grandmaster" in item.text:
-                    max_rating_span = item.find("span", class_="user-legendary")
-                    user_info["maxRating"] = (
-                        self._extract_text_or_empty(max_rating_span)
-                        if max_rating_span
-                        else ""
-                    )
-
-            # Find the <div> element with class "info"
-            info_div = soup.find("div", class_="info")
-
-            # Initialize an empty string to store the contest rating and other such info text
-            extracted_text = ""
-
-            # Check if the <div> element exists
-            if info_div:
-                # Find the <ul> element within the <div>
-                ul_element = info_div.find("ul")
-
-                # Check if the <ul> element exists
-                if ul_element:
-                    # Find all <li> elements within the <ul>
-                    li_elements = ul_element.find_all("li")
-
-                    # Loop through each <li> element and extract its text
-                    for li in li_elements:
-                        text = li.get_text(strip=True)
-                        extracted_text += text + " "
-
-            temp = extracted_text.split(":")
-
-            user_info["contribution"] = temp[2].split()[0]
-            user_info["friendsOfCount"] = temp[3].split()[0]
-            user_info["lastVisit"] = int(temp[4].split()[0]) * 3600
-            user_info["registered"] = int(temp[5].split()[0]) * 365.25 * 24 * 60 * 60
-
-            # Extract title photo and avatar
-            title_photo_div = soup.find("div", class_="title-photo")
-            user_info["titlePhoto"] = (
-                title_photo_div.find("img")["src"] if title_photo_div else ""
-            )
-            user_info["avatar"] = (
-                title_photo_div.find("img")["src"] if title_photo_div else ""
-            )
-
-            # Extracting keys
-            keys = [f"{i}".lower() for i in user_info.keys()]
-
-            # Extracting values
-
-            values = [f"{i}" for i in user_info.values()]
-
-            response = dict(zip(keys, values))
-
-            return response
+        # API Method to get user information
+        api_url = f"https://codeforces.com/api/user.info?handles={self.username}"
```

Comment:
The project's scrape-up goal is to overcome the issues faced by APIs like rate-limiting and keys/tokens. Using the APIs underneath doesn't make sense.

Also, do not start working on an issue before getting assigned.

Reply:
I apologize for the inconvenience caused, I am new to open source contributions, ao I was just trying to start contributing in different open source projects. Do let me know, if I can contribute something in future.